src/policy/fees.cpp

```diff
@@ -512,27 +433,17 @@ CFeeRate CBlockPolicyEstimator::estimateFee(int confTarget)
     return CFeeRate(median);
 }
 
-double CBlockPolicyEstimator::estimatePriority(int confTarget)
-{
-    // Return failure if trying to analyze a target we're not tracking
-    if (confTarget <= 0 || (unsigned int)confTarget > priStats.GetMaxConfirms())
-        return -1;
-
-    return priStats.EstimateMedianVal(confTarget, SUFFICIENT_PRITXS, MIN_SUCCESS_PCT, true, nBestSeenHeight);
-}
-
 void CBlockPolicyEstimator::Write(CAutoFile& fileout)
 {
     fileout << nBestSeenHeight;
     feeStats.Write(fileout);
-    priStats.Write(fileout);
```

Comment:
Until the next release is cut (which would be 3.1.0-rc1 normally), we need to continue to write out at least an empty `priStats` here (so that the on-disk versioning remains consistent). So either add a `bool hasPriStats` parameter here and conditionally write as with the reader (which we can remove after the next release), or we could bump the version to `3.1.0-beta1` in this PR.